app/scripts/config/config.js

```diff
@@ -8,6 +8,6 @@ module.exports = {
     }
   },
   apiRoot: process.env.APIROOT || 'https://example.com',
+  // recoveryPath: 'recover',
```

Comment:
Is there any reason to leave this in commented versus removing it?

Reply:
not really. This brings up a good point - I should probably add some documentation along with this PR outlining the functionality.